swap_meet/vendor.py

```diff
@@ -1,2 +1,66 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        if not inventory:
+            inventory = []
+        self.inventory = inventory
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+    
+    def remove(self, item):
+        if not item in self.inventory:
+            return False
+        else: 
+            removed_item = item
+            self.inventory.remove(item)
+            return removed_item
+
+    def get_by_id(self, item_id):
+        for item in self.inventory:
+            if item_id == item.id:
+                return item
```

Comment:
The Wave 2 requirements ask you to *explicitly* return `None`. When you don't put a return statement, you're *implicitly* returning `None`.

>If there is no matching item in the inventory, the method should explicitly return None

```python
    def get_by_id(self, item_id):
        for item in self.inventory:
            if item_id == item.id:
                return item
        return None
```